tests/testthat/test-select_helpers.R

```diff
@@ -291,6 +305,23 @@ test_that("select_helpers: .formula_list_to_named_list ", {
     .formula_list_to_named_list(list(~ "Age", `age:trt` = "interact"), var_info = tidy_mod),
     list(age = "Age", trt = "Age", grade = "Age", `age:trt` = "interact")
   )
+
+  expect_error(
+    .formula_list_to_named_list(list(age ~ "Age"), var_info = tidy_mod, type_check = is.logical,
+                                arg_name = "label")
+  )
+  expect_error(
+    .formula_list_to_named_list(list(age ~ "Age"), var_info = tidy_mod,
+                                type_check = is.logical, arg_name = "label",
+                                type_check_msg = "Age msg error")
```

Comment:
we should test if the error message contains "Age msg error"?

Reply:
good idea :)